.changeset/honest-swans-yell.md

```diff
@@ -0,0 +1,5 @@
+---
+"paris": minor
+---
+
+Update to new color organization
```

Comment:
now that we're no longer rolling out breaking changes here, this is not too bad; but in general, we _definitely_ would want a better changeset if there were breaking changes. this is the log that gets published with each release (see https://github.com/slingshot/paris/releases) so for any other consumers of this package, this would be the spot where they'd expect to see notes if anything were breaking

notably also, the changes here are not just to colors, but to the theme as a whole—we're also making adjustments to things like lighting and blurs

for now, I would at least update this to something like `Theme: Revamped variables with updated structures, and new materials/blurs`

Reply:
(even though we don't have a ton of known consumers right now — paris is still seeing a decent amount of downloads at NPM and we want to be good stewards of any library that we choose to make open source so that others can use them reliably if they want to)